new_app/js/actions/AppActions.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Created by omelvin on 5/22/15.
```

Comment:
git already handles creation/updated at history. probably not necessary? thoughts?